cpp/src/gandiva/engine.h

```diff
@@ -93,6 +93,9 @@ class GANDIVA_EXPORT Engine {
   /// the main module.
   Status LoadPreCompiledIR();
 
+  // load external pre-compiled IR modules from LLVMIRStore
```

Comment:
`LLVMIRStore` doesn't exist anymore, does it? Can you update this docstring?

Reply:
Good catch. Updated.